### PR TITLE
Scripts: async, defer

### PR DESCRIPTION
**Pull request by larissaiurk**

### CODE REVIEW

**Comment by odsantos** [comment]
Hi @larissaiurk,

Still available for this task?
There is a conflict here.

Please, let me know if you have not time, and I might be able to resolve it.

Thanks,
Osvaldo.

**Comment by nazarepiedady** [comment]
@Peruibeloko, @jonnathan-ls, do you have time to see what is possible to do in relation to this pull request?

**Comment by nazarepiedady** [comment]
@Peruibeloko, could you help to fix the conflicts?

**Review by jonnathan-ls — APPROVED** [review]
LGTM

Adicionei ajustes simples (0dfdf2503ca1d5d7b82e0514af1dcc6bb21c68bc) e resolvi o conflito de merge (b0b6ae0ba95c5df71fb4e838ebe1056f5e6f0d71)

**Comment by jonnathan-ls** [comment]
> ... could you help to fix the conflicts, update, and approve it?

During the review, I added simple adjustments (including the suggestion requested by @Peruibeloko) and also conflict resolution @nazarepiedady

File: 2-ui/5-loading/02-script-async-defer/article.md [hunk 1]
```diff
@@ -1,163 +1,163 @@
...
+Quando o navegador carrega o HTML e encontra uma tag `<script>...</script>`, ele não pode continuar construindo o DOM. Ele deve executar o script no momento exato que o encontra. O mesmo acontece com scripts externos `<script src="..."></script>`: o navegador deve aguardar o download do script, executar o script baixado e só então pode processar o resto da página.
 
-That leads to two important issues:
+Isso nos leva a duas questões importantes:
 
-1. Scripts can't see DOM elements below them, so they can't add handlers etc.
-2. If there's a bulky script at the top of the page, it "blocks the page". Users can't see the page content till it downloads and runs:
+1. Os scripts não conseguem ver os elementos da DOM abaixo deles, então não conseguem manipular eventos, etc.
+2. Se houver um script volumoso no topo da página, ele vai "bloqueia a página". Os usuários não podem ver o conteúdo da página até que ele faça o download e execute:
```
[the remark this answers is not in the document]
Comment (jonnathan-ls) [reply 2-ui/5-loading/02-script-async-defer/article.md]: Ajustado

File: 2-ui/5-loading/02-script-async-defer/article.md [hunk 2]
```diff
@@ -129,13 +129,17 @@ Eles não esperam um pelo outro. Qualquer um que carregue primeiro (provavelment
...
 ```
 
-## Scripts dinâmicos
- 
-Existe mais uma maneira importante de adicionar um script à página.
+```smart header="O atributo `async` é apenas para scripts externos"
+Assim como `defer`, o atributo `async` é ignorado se a tag `<script>` não tiver `src`.
+```
+
+## Dynamic scripts
```
Comment (nazarepiedady) [inline 2-ui/5-loading/02-script-async-defer/article.md:140]: It was not translated yet.

File: 2-ui/5-loading/02-script-async-defer/article.md [hunk 3]
```diff
@@ -129,13 +129,17 @@ Eles não esperam um pelo outro. Qualquer um que carregue primeiro (provavelment
...
+
+There's one more important way of adding a script to the page.
```
Comment (nazarepiedady) [inline 2-ui/5-loading/02-script-async-defer/article.md:142]: It also was not translated yet.

**Review by nazarepiedady — CHANGES_REQUESTED** [review]
It is good, but there are some little changes needed.

**Comment by nazarepiedady** [comment]
@jonnathan-ls, there are some changes that need to be translated.

File: 2-ui/5-loading/02-script-async-defer/article.md (hunk 2 above)
Comment (jonnathan-ls) [reply 2-ui/5-loading/02-script-async-defer/article.md]: Done

File: 2-ui/5-loading/02-script-async-defer/article.md (hunk 3 above)
Comment (jonnathan-ls) [reply 2-ui/5-loading/02-script-async-defer/article.md]: Done

**Comment by jonnathan-ls** [comment]
@nazarepiedady Thanks for watching, pending translation added

/done

**Review by nazarepiedady — APPROVED** [review]
It is good to be applied.

**Pull request merged**